### DISCUSSION TITLE
I miss the old forums  :-(

### DISCUSSION THREAD

**Discussion by wayner9** [OpenSageTV/Community#75] [General]
I have given this new forum a chance but I just find it is not nearly as useful as the old forums.  I would enjoy going to the forums and see what was happening.  These forums, while looking more up-to-date, just don't seem as functional.  

It is kind of a bummer

**Comment by checkbin99** [comment]
I second that.  The new site just doesnt provide the same ease of use as the old one.  Very sad to see that one go.

**Comment by emveepee** [comment]
Is it the really the forum software you miss?  For me the Sage TV forum used to be a vibrant source of information about PVR in general, and web searching for general topics still bring back posts from the old forum.  Now with no developers or development, no new features,  few new users, long time users leaving etc it is not the same, which is why I stopped visiting daily. 

There doesn't appear to be a way to "Make SageTV Great Again" but the good thing is the software and platform are stable just not exciting to read about.

**Reply by wayner9** [reply]
I miss the community as well, but that had petered out over the last decade.  But the old forum software had denser info -  in a page or two you could see everything that was going on.

**Reply by LoisGNS** [reply]
I wonder if there are better options? 
Reddit isn't suitable, as it's just a running batch of messages, without categories, file storage, etc.
GroupsIO has some useful features: file storage (but limited for free groups, and premium groups cost $20 per month, or $.04 per user per month, whichever is greater); subforums are supported, but only for premium groups.
Google Groups - there actually exists one for Sage (it's inactive), and while it's possible to attach files to messages, I don't see a file repository available, although I belong to some groups that also have their own Gdrive locations, so something along those lines might be doable.
Discord - I've used it a bit, but not enough to be very conversant about its features & limitations. I don't see any place there to store files.

**Comment by KryptoNyte-Sage** [comment]
I miss both, but on the topic of the forum, although I'm thankful to have a place to continue on with Sage, I really miss not being able to readily see the new activity.  I've even wondered if there is a way to disable the "reply to a specific post" so that every piece of activity turns the forum topic white again, and a user can always find new activity at the bottom.

**Comment by KryptoNyte-Sage** [comment]
I want to thank those that setup this Github forum, regardless.  Without it, we'd have nothing.
